conda-recipe/meta.yaml

```diff
@@ -29,20 +29,17 @@ requirements:
     - humanize
     - numpy
     - pytz
+    - boltons
 
 test:
   requires:
     - nslsii_dev_configuration
 
-  # Python imports
   imports:
+    # Import all the packages
     - metadatastore
-    - metadatastore.commands
```

Comment:
lmao, ok.  I have added **all** the non-test modules to the meta.yaml.  Let the shaming of @tacaswell commence!